articles/install-guide/setup/vs-editors/vs-2017.md

```diff
@@ -1,28 +1,41 @@
 ---
-title: Installing the Q# development environment for VS 2017
+title: Installing the Q# development environment for VS 2017 or later
 author: cgranade
 uid: microsoft.quantum.install.vs-2017
 ms.author: christopher.granade@microsoft.com
 ms.date: 7/31/2018
 ms.topic: article
 ---
 
-# Installing the Quantum Development Kit for Visual Studio 2017 #
+# Installing the Quantum Development Kit for Visual Studio #
 
-## Installing Visual Studio 2017 (Windows 10 only)
+The Quantum Development Kit can be used with Visual Studio (2017 or later), Microsoft's enterprise-grade development environment for Windows.
+
+Below, you will first find instructions for installing the Quantum Development Kit extension directly--follow these if you already have Visual Studio.
```

Comment:
I think it might be more concise to simplify this section into:
Installing Visual Studio 2019 (I think people will naturally skip/skim this if they already have it installed)
Installing the extension
Validating the install